### ISSUE TITLE
Intellisense isn't working after my vscode updated to v1.3

### ISSUE THREAD

**Issue by tobia88** [dotBunny/VSCode#116]
There's no intellisence after my vscode updated, please help

**Comment by Taugeshtu** [comment]
There are issues with Unity and C# support in latest VSCode C# extension. My buddy's using [this legacy C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.omnisharp).

[Issue related to C# extension on github](https://github.com/OmniSharp/omnisharp-vscode/issues/269#issuecomment-232194196).

**Comment by 777777716** [comment]
My Version is 1.3.1, but intellisence dosen‘t work, it's also hanpend in 1.3.0.

**Comment by iiley** [comment]
I have same issue here, with VSCode 1.3.1 & legacy C# extension, Mac OS.

**Comment by frankprogrammer** [comment]
I am using 1.4.0 Insiders version and it is working fine on my mac. Have you tried installing Mono? http://www.mono-project.com/download/  That fixed it for me when I was initially setting things up a month ago.

**Comment by reapazor** [comment]
Indeed it seems to have been limited to the 1.3.x chain, as the Visual Studio Code team has already commented they were going to fix it for the next major release ... and it looks like they have.

**Comment by iiley** [comment]
@frankprogrammer do you use legacy C# extension with 1.4.0 insiders version?

**Comment by frankprogrammer** [comment]
Legacy C# Support 0.3.3
Debugger for Unity 1.0.3

This is the way I set up my system before the regular VS Code added tab support. 
http://pastebin.com/w799Mb8Z

I am guessing though that this now should work under the regular VS Code without having to modify VSCode.cs

**Comment by iiley** [comment]
Thanks frank, your suggestion works on my win10 now, i will try it later on my Mac.

**Comment by 777777716** [comment]
I found the way to fix it last night!
I install Legacy C# Support and C# for Visual Studio Code, then my intellisense can't work.
![image](https://cloud.githubusercontent.com/assets/7316089/17082096/e80e7d5e-51a3-11e6-8176-4cc042d4a813.png)
![image](https://cloud.githubusercontent.com/assets/7316089/17082092/d68ba2a0-51a3-11e6-8dfc-a4f17f4203a6.png)
So, I try to uninstall  C# for Visual Studio Code, my intellisense restart working. 
I hope this way can help you gays.

**Comment by iiley** [comment]
Very strange, i can not get intellisense on my Mac with VS Code insiders version but it works on my windows. I have not install C#, only Legacy C# support & Debugger for Unity.

**Comment by lishali12345** [comment]
It's worked for me. This is my IDE and versions:
1. Mac OSX EI Capitan (Version 10.11.6)
2. Unity (Version 4.6.9f1)
3. Visual Studio Code (Version 1.3.1 )
4. C# for Visual Studio Code (Version 1.3.0) (This version has fixed the problem that which make the Unity or Mono development on Mac OS X not working well, just like @777777716 said, the official tell us to switch the C# extension to the Legacy one which bundled with VS Code. But now, the Version 1.3.1 has released, we just only should do is update the C# extension to this latest version, it's done very well.)
5. MonoFramework-MDK-4.4.0.142.macos10.xamarin.universal

That's all. But notice.

When we use the menu item " Open C# Project In Code", the Code will only open the project folder, then we should choose a C# source code file to open it. Then the OmniSharp will start to work, we can choose open which .sln file in the project folder, when we choosed a sln file to open (we should choose the sln file which file name with "-csharp.sln" suffix), the OmniSharp will download the OmniSharp Server components from the internet, just be patient, wait the downloading finished. Then the Intellisense can worked very well.

**Comment by iiley** [comment]
Hi @lishali12345 I just make my VSCode Version 1.4.0-insider works with what you suggested, that is nice.

What i have done is:
1. Uninstall the Legacy C# support.
2. Install the C# 1.3.0 support
3. Download and install .net core from https://www.microsoft.com/net/core#macos
4. Download and install MonoFramework-MDK-4.4.0.142.macos10.xamarin.universal from http://download.mono-project.com/archive/4.4.2/macos-10-universal/MonoFramework-MDK-4.4.2.11.macos10.xamarin.universal.pkg

Finally works on my mac now. Thanks for sharing.

**Comment by lishali12345** [comment]
@iiley Glad to help.

**Comment by pavel1860** [comment]
hi.
I'v tried the solutions on this thread but the nothing worked. I have no intellisense and the omnisharp flame icon is just displaying "Starting...".

**Comment by reapazor** [comment]
I'm wanting to close this this mainly because the VS Code team has resolved their issues, so long as you uninstall/disable the legacy c# extension and reenable the newer one it should work just fine. 

What does the omnisharp log report as a problem?

**Comment by McDirty** [comment]
Following the above steps to fix this problem, I'm still running into issues like @pavel1860 is having. I am stuck with the following: 

![camerafollow_cs_-_instrument_unity_research_and_adammccurdy_ _adammccurdy_adammccurdy_ _-zsh_ _181x94](https://cloud.githubusercontent.com/assets/3011035/19658808/41f85b8c-99de-11e6-9614-07b2c88b8798.png)

I have Installed, .net core, mono 4.6.x, made sure legacy c# support was uninstalled in VScode, and Omnisharp C# is installed. 

TL;DR no intellisense, stuck at "Starting"

**Comment by Disar** [comment]
.net core requires me to install visual studio 2015. Which defeats the purpose of having visual studio code. I'm a bit baffled.

Legacy extension is gone, and the last omnisharp requires .net core. There is no winning here. Any other tricks I could use?

**Comment by Taugeshtu** [comment]
Disar, try this link: http://www.microsoft.com/net/core#windowscmd

**Comment by Disar** [comment]
Installed it, nothing happens. =/ when hitting ctrl+space is just shows the available keywords/variable names from the script itself. It doesn't actually process anything else.

**Comment by Disar** [comment]
It automagically started working, perhaps the .net core wasn't done doing its thing.

**Comment by reapazor** [comment]
It looks like the folks over at M$ have figured out their c# core stuff and have simplified things. Removing the legacy stuff, things should just work now.
